package.json

```diff
@@ -14,28 +14,32 @@
     "2d",
     "mobile"
   ],
+  "author": "Ali Shakiba",
   "license": "MIT",
   "repository": {
     "type": "git",
     "url": "git://github.com/shakiba/planck.js.git"
   },
-  "main": "dist/planck.js",
-  "types": "index.d.ts",
-  "author": "Ali Shakiba",
   "contributors": [
     "Erin Catto",
     "Ali Shakiba",
     "Oliver Zell"
   ],
-  "dependencies": {
-    "stage-js": "^0.9.4"
+  "type": "module",
+  "types": "./index.d.ts",
+  "main": "./dist/planck.umd.cjs",
+  "module": "./dist/planck.js",
+  "browser": "./dist/planck.cjs",
+  "jsdelivr": "dist/planck.umd.cjs",
+  "unpkg": "dist/planck.umd.cjs",
+  "exports": {
+    ".": {
+      "types": "./index.d.ts",
```

Comment:
I'm not entirely sure if index.d.ts is correct here (index.d.ts is for und, I think ./dist/planck.d.ts is correct here, but as I said I'm not sure)